requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+black
```

Comment:
I am not sure we want to add this file. Dev requirements are already specified through `setup.py` here https://github.com/MasoniteFramework/masonite/blob/4.0/setup.py#L188

You can install those by doing: `pip install 'masonite[test]'` 👉 that's what the makefile is doing when doing `make init` in a fresh development setup 😉.

Reply:
So I think moving `black` in those extra dev dependencies would be better (pytest is already there), instead of creating this requirements-dev.txt file.

Reply:
Aah yeah makes sense... will do

Reply:
> So I think moving `black` in those extra dev dependencies would be better (pytest is already there), instead of creating this requirements-dev.txt file.

Hmm but if we are separating the tests from the framework should we also not remove the pytest and other development dependencies from the setup.py?  
as they are only used for development and testing why would we include them when deploying for production purposes if they are not used ata all?

Si like you mentioned using `pip install 'masonite[test]'` would be the recommended method to install all development deps. Thus keeping the Framework cleaner.